packages/jest-mock-apollo/src/factory.ts

```diff
@@ -0,0 +1,81 @@
+import {buildSchema} from 'graphql';
```

Comment:
`factory` is not a useful name for a file — factory of what? IMO the file name should represent what its purpose is, not how it accomplishes that